andela/ah-frontend-odin
# #161340138 Users should be able to reset their password

### What does this PR do?
This PR allows users to be able to reset their passwords once they have forgotten
### Any Background context?
1. This only works for registered users only
### How can this be manually tested?
1. Just feed in the email address that you used to register for Authors Haven
2. Hit `Send me link` button and check your email
3. Click the link that you received on your email
4. Provide your new password and hit `Reset Password` button and you are done, you can go ahead and login using the new password

### Screenshots for the feature
1. Where users feed in their email address to request reset link
<img width="1438" alt="screen shot 2018-12-06 at 01 32 44" src="https://user-images.githubusercontent.com/20398831/49598011-c32d5380-f98e-11e8-9f99-ce18fb717f8e.png">


2. Where users then feed in their new passwords
<img width="1437" alt="password" src="https://user-images.githubusercontent.com/20398831/49464986-6d7d6d80-f80d-11e8-99be-e42b906ecbd5.png">
